scss/bitstyles/atoms/badge/badge.stories.mdx

```diff
@@ -1,117 +1,133 @@
 import { Canvas, Meta, Story } from '@storybook/addon-docs';
+import Badge from './Badge';
 import icons from '../../../../assets/images/icons.svg';
 
-<Meta title="Atoms/Badge" />
+<Meta title="Atoms/Badge/Overview" component={Badge} />
+
+<Canvas>
+  <Story id="atoms-badge--base" />
+  <Story id="atoms-badge--brand-1" />
+  <Story id="atoms-badge--brand-2" />
+  <Story id="atoms-badge--danger" />
+  <Story id="atoms-badge--warning" />
+  <Story id="atoms-badge--positive" />
+</Canvas>
+
+<Canvas>
+  <Story id="atoms-badge--base-small" />
+  <Story id="atoms-badge--brand-1-small" />
+  <Story id="atoms-badge--brand-2-small" />
+  <Story id="atoms-badge--danger-small" />
+  <Story id="atoms-badge--warning-small" />
+  <Story id="atoms-badge--positive-small" />
+</Canvas>
 
 # Badge
 
 Display tags, filters, or categorization.
 
-<Canvas>
-  <Story name="Badge">
-    {`
-      <span class="a-badge a-badge--grayscale u-h6 u-font-medium">Urgent</span>
-      <span class="a-badge a-badge--grayscale u-h6 u-font-medium">In progress</span>
-    `}
-  </Story>
-</Canvas>
+## Variants
+
+Badges are available in a range of color variations, and two size variations (when using the default configuration — see [Customization](#customization) below for details on how to change that).
+
+Color variants:
+
+- grayscale (default)
+- brand-1
+- brand-2
+- danger
+- warning
+- positive
 
-By default `gray`, `brand-1`, `brand-2`, and `danger` colors are available (see [Customization](#customization) section below). Here, they’re displayed in a list (there are often multiple tags or filters so this is very common).
+Size variants:
+
+- base (default)
+- small
+
+All badges are displayed as `small` badges on small screens.
+
+## “Extra” area
```

Comment:
What about "prepend" for things on the left and "append" for things on the right?